rest/route/service.go

```diff
@@ -126,6 +126,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/projects").Version(2).Get().Wrap(checkUser).RouteHandler(makeFetchProjectsRoute(sc))
 	app.AddRoute("/projects/{project_id}").Version(2).Get().Wrap(checkUser, addProject, checkProjectAdmin, viewProjectSettings).RouteHandler(makeGetProjectByID(sc))
 	app.AddRoute("/projects/{project_id}").Version(2).Patch().Wrap(checkUser, addProject, checkProjectAdmin, editProjectSettings).RouteHandler(makePatchProjectByID(sc, env.Settings()))
+	app.AddRoute("/projects/{project_id}/repotracker").Version(2).Post().Wrap(checkUser, addProject).RouteHandler(makeRunRepotrackerForProject(sc))
```

Comment:
This should `checkProjectAdmin`.

Reply:
Will that work for the smoke user?

Reply:
Update it doesn't work for smoke tests. I'm in favor of allowing triggering the repotracker to be available to all users since we won't document it and it isn't a security risk, but we could alternatively add the smoke user to evergreen's project admins? 🤔

Reply:
Ah yeah I can't think of a specific reason not to allow all users to use this endpoint. Let's leave it as is.